backend/src/index.js

```diff
@@ -28,40 +28,87 @@ const s3Client = new S3Client({
 });
 
 /**
- * Endpoint for fetching CSV data from S3.
+ * Endpoint for fetching project data and converting it to CSV format.
  * @route GET /api/csv
- * @returns {Object[]} Array of objects containing parsed CSV data
- * @returns {Object} response.data - Each row from the CSV as an object with column headers as keys
- * @throws {Error} 500 - If CSV fetching or parsing fails
+ * @returns {Object[]} Array of objects containing parsed project data in CSV format
+ * @throws {Error} 500 - If data fetching or processing fails
  */
 app.get("/api/csv", async (req, res) => {
   try {
     const command = new GetObjectCommand({
-      Bucket: bucketName,
-      Key: "onsTechDataAdoption.csv",
+      Bucket: tatBucketName,
+      Key: "new_project_data.json",
     });
 
     const signedUrl = await getSignedUrl(s3Client, command, { expiresIn: 300 });
 
-    // Fetch the CSV data using the signed URL
+    // Fetch the JSON data using the signed URL
     const response = await fetch(signedUrl);
-    const csvText = await response.text();
-
-    // Parse the CSV data and filter out empty or incomplete entries
-    Papa.parse(csvText, {
-      header: true,
-      complete: (results) => {
-        // Filter out empty row
-        const filteredData = results.data.filter(entry => Object.keys(entry).length > 1);
-        res.json(filteredData);
-      },
-      error: (error) => {
-        logger.error("Error parsing CSV:", { error });
-        res.status(500).json({ error: "Failed to parse CSV data" });
-      },
+    const jsonData = await response.json();
+
+    // Transform JSON data to CSV format
+    const transformedData = jsonData.projects.map(project => {
```

Comment:
I think it'd be cleaner if the transformation was handled by a utility function rather than done in-line in an arrow function